packages/@ourworldindata/utils/src/MarkdownTextWrap/MarkdownTextWrap.tsx

```diff
@@ -296,9 +296,9 @@ export class IRDetailOnDemand extends IRElement {
     }
     toHTML(key?: React.Key): JSX.Element {
         return (
-            <DoDWrapper key={key} term={this.term} category={this.category}>
+            <a key={key} className="dod-span" data-id={this.term}>
```

Comment:
Looking at the gdoc, it seems we're getting rid now of the category/term distinction?

Reply:
Yeah I don't think the category term thing really made an impact in the end. (I'll double check with the authors on that.)

If we have two different types of poverty that we want to define we can just treat them as separate terms: `poverty-growth` `poverty-health`